axlearn/audio/asr_decoder.py

```diff
@@ -28,6 +28,43 @@
 from axlearn.common.utils import Nested, Tensor
 
 
+def _is_valid_ctc_seq(
+        paddings: Tensor,
+        target_labels: Tensor,
+        target_paddings: Tensor) -> Tensor:
+    """Returns for per example sequence if it passes validity check.
+
+    Note that `optax.ctc_loss` returns logeps (default to 1e5) if the
```

Comment:
Can we give a reference link to optax implementation on this?

Reply:
Oh i mean add this reference in code comment explicitly.

Reply:
Sorry I think there is no explicit mention in optax how to deal with inaccessible path, so I prefer not to mention specific line in the code.